lms/djangoapps/courseware/views/views.py

```diff
@@ -1892,7 +1894,11 @@ def get_public_video_cta_button_urls(self, course, catalog_course_data):
             },
             utm_params
         )
-        return learn_more_url, enroll_url
+        user = self.request.user
+        go_to_course_url = (get_learning_mfe_home_url(course_key=course.id,
```

Comment:
I think it might be slightly cleaner if we always include the "go to course url", but also add a "user enrolled in course" flag in the context, then the check in the template is on the "user enrolled in course" flag. Then what we're actually trying to do becomes clearer in the template.